### ISSUE TITLE
Missing typings in data_enhancement

### ISSUE THREAD

**Issue by OskarGroth** [firecmsco/firecms#586]
In a fresh Quick Start project I get the following error:

```
Could not find a declaration file for module '@firecms/data_enhancement'. '/Users/oskar/backend/cms/node_modules/@firecms/data_enhancement/dist/index.es.js' implicitly has an 'any' type.
  There are types at '/Users/oskar/backend/cms/node_modules/@firecms/data_enhancement/dist/index.d.ts', but this result could not be resolved when respecting package.json "exports".
The '@firecms/data_enhancement' library may need to update its package.json or typings.
```

Probably related to: https://github.com/firecmsco/firecms/issues/500

**Comment by OskarGroth** [comment]
It seems there is a typo:

<img width="607" alt="Screenshot 2023-11-07 at 16 04 15" src="https://github.com/firecmsco/firecms/assets/1417019/8e41f6e8-f84e-416f-960f-8dfdb0ba0052">


`src` should not be here. Removing it fixes the issue

**Comment by fgatti675** [comment]
just changed an released version 1.2.1 :)
